src/example/App/VariableText.js

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import createReactClass from 'create-react-class';
 import {shouldComponentUpdate} from 'react/lib/ReactComponentWithPureRenderMixin';
```

Comment:
well, when we switch to es6 classes we just need to use `PureComponent`, so its easily fixable